### DISCUSSION TITLE
Support for newer versions of FRR

### DISCUSSION THREAD

**Discussion by pg** [Kidel/Kathara#210] [General]
It seems the docs and available resources (Docker images, etc.) here are built on FRR 7. Is there a plan or desire to move to FRR 8? Alternatively, is there anything I can do on my end to move to FRR 8 while still using Kathara? Would it just work to update `frr-7` to `frr-8` in https://github.com/KatharaFramework/Docker-Images/blob/master/debian10/frr/Dockerfile?

**Comment by tcaiazzi** [comment]
Hi @pg, 

yes we planned to move to FRR8, but before doing that we need to ensure that all our network scenarios still work. 

Yes you can update the Dockerfile with `frr-8` and rebuild the image. 

Let me know if you need any help, 
Tommaso
